ui/components/ClusterOverview/ClusterCard.tsx

```diff
@@ -218,6 +247,69 @@ export function ClusterCard({
                         </DataListItemRow>
                       </DataListItem>
                     )}
+                    {isReconciliationPaused && (
+                      <DataListItem aria-labelledby={`reconciliation-warning`}>
```

Comment:
The `messages` array already contains a condition for `ReconciliationPaused`. Can we use that instead of adding an additional row? It results in the generated row plus the one that Strimzi gives us for `ReconciliationPaused`.

![image](https://github.com/user-attachments/assets/30eb4af4-5a67-4995-89d0-1770e000e876)

Reply:
fixed